client/src/assets/partials/lesson.html

```diff
@@ -27,6 +27,11 @@ <h3>Level {{selectedLevel}} not yet mastered</h3>
         </div>
       </div>
 
+      <div class="lesson-incorrect-chars">
+        <div class="char" ng-repeat="char in lesson.sortedIncorrectChars track by $index"style="width:{{char.percent}}%; background:rgba(227, 69, 9, {{char.opacity}})">
```

Comment:
We should avoid setting such a specific color here as it is smelly and hard to find since it is so separated from our styles.

Instead we should _find a way_ to accomplish the same result via more maintainable code.  (ask me)